docs/types.rst

```diff
@@ -49,44 +49,47 @@ Class that contains the different names used to represent hard forks on the Ethe
         Constantinople = 'Constantinople'
         Metropolis = 'Metropolis'
 
-Miscellaneous
--------------
+EthPM
+-----
 
-Address
-~~~~~~~
+ContractName
+~~~~~~~~~~~~
 
-Any bytestring representing a canonical address.
+Any string conforming to the regular expression ``[a-zA-Z][a-zA-Z0-9_]{0,255}``.
 
 .. code-block:: python
 
-    Address = NewType('Address', bytes)
+    ContractName = NewType('ContractName', str)
 
-AnyAddress
-~~~~~~~~~~
+URI
+~~~
 
-Any of Address_, HexAddress_, ChecksumAddress_.
+Any string that represents a URI.
 
 .. code-block:: python
 
-    AnyAddress = TypeVar('AnyAddress', Address, HexAddress, ChecksumAddress)
+    URI = NewType('URI', str)
 
-BlockNumber
-~~~~~~~~~~~
+EVM
```

Comment:
I've been wondering if we want to start calling it `ETH`?

Reply:
I intentionally chose EVM over ETH for this case because these are concepts from the Ethereum Virtual Machine (mostly).  

One could argue that `ChecksumAddress`, `HexAddress`, and `AnyAddress` belong under the higher level `eth` namespace (if we were to create it), since they are not actualy EVM objects (since the EVM only works with bytes addresses).